examples/fargate/app/Dockerfile

```diff
@@ -2,8 +2,7 @@ FROM node:lts AS builder
 
 WORKDIR /app
 
-COPY package.json yarn.lock ./
-COPY app ./
+COPY package.json yarn.lock index.ts ./
```

Comment:
This way the docker image only changes when the `app` folder changes not the entire `fargate` folder.